.github/dependabot.yml

```diff
@@ -21,3 +21,6 @@ updates:
     labels:
       - "CherryPick"
       - "dependencies"
+      - "6.13.z"
+      - "6.12.z"
+      - "6.11.z"
```

Comment:
this is something i've been thinking about. if we wrote pre and post-branching scripts. this would be one of the things in the post-branching script

Reply:
@JacobCallahan Is that something you would like to track as a task in some issue tracker?

Reply:
For now and until we have the scripted branching process, this has to be maintained manually like any other maintenance we are doing from versioning pov.

Reply:
Sure, let's leave scripting for later. Could we have all these places at least documented in the robottelo docs as the first step?

Reply:
This looks like a non-blocking comment and is not specific to change for now. @ogajduse let me know if you disagree with this.  Please create Jira or GitHub issue for this